### ISSUE TITLE
(Linux) "wd_export_static_named_directories:hash:4: no such directory name" when path includes spaces

### ISSUE THREAD

**Issue by jfinas** [mfaerevaag/wd#77]
warp point: `test  ->  /tmp/my dir`

Using `wd test` to jump to that directory works to be fine, but invocations to wd print an error.

```
jean@jfinas-xps15 ~ % wd test
wd_export_static_named_directories:hash:4: no such directory name: dir
jean@jfinas-xps15 /tmp/my dir % 
```
The corresponding entry in `~/.warprc` looks like this:
```
dotfiles:~/git/dotfiles
test:/tmp/my dir
```

**Comment by AtkLordOverAll** [comment]
Will be fixed by https://github.com/mfaerevaag/wd/pull/76

**Comment by mfaerevaag** [comment]
@jfinas Can you confirm the the @AtkLordOverAll 's PR solves it for you?

**Comment by jfinas** [comment]
sorry for replying so late, i struggled to figure out how to actually test that(rather new to this...)

but yes, it seems to fix the issue for me.

**Comment by alpha-tango-kilo** [comment]
Can be closed now as #76 was merged
